### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by zultralord**
Updated the organization of the Readme file to include a more robust and structured presentation of the projects summary information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
-### Sending a Pull Request
+### Bug Reporting
+We are using [GitHub Issues](https://github.com/DevChatter/devchatterbot/issues) for our bug reports. Make sure to choose the correct label when creating the issue. Please include details of what you noticed as well as instructions for reproducing the issue. Please provide plenty of details to make it easier for us!
+
+### Sending Pull Requests
 Details coming soon.
+Have I mentioned that you're awesome for even considering contributing to this project? Thanks!
+
+## Contributors
+* That One Guy ([ThatOneGuy](https://github.com/thatoneguy))
```
Comment (benrick) [inline README.md]:
```suggestion
* AridTag ([AridTag](https://github.com/AridTag))
* Dalton ([DCurtin](https://github.com/DCurtin))
* Justin ([jdphenix](https://github.com/jdphenix))
* Jake ([asyncawake](https://github.com/asyncawake))
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,51 @@
...
+
+## Acknowledgments
+* TwitchLib - https://github.com/twitchlib
```
Comment (benrick) [inline README.md]: Move this to dependencies.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,50 @@
-# DevChatter Chat Bot
+<p align="center">
+  <a href="#" alt="Devchatterbot" width="200">img</a>
```
Comment (benrick) [inline README.md]:
```suggestion
  <a href="#" alt="Devchatterbot" width="200"><img src="https://www.devchatter.com/assets/css/images/DarkBack-Logo.png"></a>
```

File: README.md [hunk 4]
```diff
@@ -1,19 +1,50 @@
...
+Have I mentioned that you're awesome for even considering contributing to this project? Thanks!
+
+## Contributors
+* AridTag ([AridTag](https://github.com/AridTag))
+* Dalton ([DCurtin](https://github.com/DCurtin))
+* Justin ([jdphenix](https://github.com/jdphenix))
+* Jake ([asyncawake](https://github.com/asyncawake))
+
+## License
+This project is licensed under the ??? License - see the [LICENSE.md](LICENSE.md) file for details
```
Comment (benrick) [inline README.md]:
```suggestion
This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details
```

Comment (benrick) [inline README.md]:
```suggestion
This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details
```

File: README.md (hunk 1 above)
Comment (zultralord) [reply README.md]: Complete

File: README.md (hunk 2 above)
Comment (zultralord) [reply README.md]: Complete

**Pull request closed without merging**